server/cluster/cluster_worker.go

```diff
@@ -196,6 +196,16 @@ func (c *RaftCluster) HandleReportSplit(request *pdpb.ReportSplitRequest) (*pdpb
 	originRegion := proto.Clone(right).(*metapb.Region)
 	originRegion.RegionEpoch = nil
 	originRegion.StartKey = left.GetStartKey()
+
+	for _, region := range []*metapb.Region{left, right} {
```

Comment:
We will do some checks before calling `HandleRegionHeartbeat` in a normal Region heartbeat process, should we do the same thing here?

Reply:
I think it does not need to checker, it will call HandleRegionHeartbeat too.